digiwf-gateway/src/main/resources/logback-spring.xml

```diff
@@ -90,28 +91,10 @@
         </appender>
 
         <!-- Logger -->
-        <logger name="io.muenchendigital.digiwf.gateway"
```

Comment:
warum hast du das rausgenommen?

Reply:
ich sehe hier keine Einstellung auf root ebene. deswegen bin ich da bisschen verwirrt.

Reply:
Hab alles mit info entfernt, weil das ja über: https://github.com/it-at-m/digiwf-core/blob/d6e344dec1c6b557bdb2033f87c1146e73e0bb8a/digiwf-gateway/src/main/resources/logback-spring.xml#L101 angezogen werden sollte oder?

Reply:
habe ich übersehen. mein fehler. :-)